### PR TITLE
Bugfix: Fix LIGHTS warning on HMD

### LINKED ISSUE #125: Bug: “LIGHTS” text will not go away on HMD when lights turned off
# Prerequisites

Please answer the following questions for yourself before submitting an issue. 

- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this [issue](https://github.com/Project-Hatchet/public_h-60/issues?q=is%3Aissue+label%3Abug+) has not already been filed
- [x] I checked the [FAQ](https://discord.gg/4AYEfFD) and known bugs on the discord
  

# **RPT log file:**

- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file. An instruction to find your RPT files can be found [here](https://community.bistudio.com/wiki/Crash_Files#Arma_3).

# Bug Report

## **Description:**

LIGHTS text on HMD will not go away when you turn lights off.

### **Expected Behavior**

LIGHTS text to go away

### Steps to Reproduce the Issue

1. Enable HMD
2. Turn on lights
3. Turn off lights
4. Notice lights text is still there

### **Screenshots**

Will add later
 (Unable to use laptop, cant add pic rn)

### PR DESCRIPTION
**Pull request by BroBeansCPG**
**When merged this pull request will:**
fix #125

### CODE REVIEW

**Review by YannikH — APPROVED** [review]
Great stuff!

File: addons/uh60_misc/functions/fnc_perFrame.sqf [hunk 1]
```diff
@@ -16,6 +16,10 @@ if (inputAction "headlights" > 0 && !vtx_uh60_misc_btn_hold) then {
     };
 };
 
+if (!vtx_uh60_misc_landingLightOn && !(_vehicle animationPhase "Switch_lights_position" > 0.5) && !(_vehicle animationPhase "Switch_lights_cockpit" > 0.5)) then {
```
Comment (YannikH) [inline addons/uh60_misc/functions/fnc_perFrame.sqf]: does this need to run on every frame or can it run once per second

**Review by YannikH — CHANGES_REQUESTED** [review]
Question about where this code is run

File: addons/uh60_misc/functions/fnc_perFrame.sqf (hunk 1 above)
Comment (YannikH) [reply addons/uh60_misc/functions/fnc_perFrame.sqf]: Or can we maybe use an eventhandler

Comment (ampersand38) [reply addons/uh60_misc/functions/fnc_perFrame.sqf]: The logic should be handled in vtx_uh60_misc_fnc_toggleLights, which is run by all the light switches as well as the basegame input.

Comment (BroBeansCPG) [reply addons/uh60_misc/functions/fnc_perFrame.sqf]: I think i got bogged down in things not working properly and forgot to move it back to toggleLights fnc. 

Resolved & ready to merge

**Pull request closed without merging**